src/main/java/com/islandcollaborative/creativeexchange/configs/ModelAttributeConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.islandcollaborative.creativeexchange.configs;
+
+import com.islandcollaborative.creativeexchange.models.AppUser;
+import com.islandcollaborative.creativeexchange.repositories.AppUserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ModelAttribute;
+
+import java.security.Principal;
+
+@ControllerAdvice
```

Comment:
I assume this is the piece of code that adds the principal into every page if available?